MOBILE_CLIENT/COMMON/_HIGH/REGEX_DOS/description.md

```diff
@@ -0,0 +1,98 @@
+Regular Expression Denial of Service (ReDoS) is a security vulnerability that occurs when user input is utilized to construct a regular expression. In the presence of a carefully crafted regex pattern, the application may expend a substantial amount of resources, potentially resulting in a denial of service.
+
```

Comment:
Can you mention the Evil Regexes

Reply:
I think that would be specific to each case